core/field.js

```diff
@@ -527,7 +527,8 @@ Blockly.Field.prototype.setValue = function(newValue) {
  * @param {!Event} e Mouse down event.
  * @private
  */
-Blockly.Field.prototype.onMouseDown_ = function(e) {
+Blockly.Field.prototype.onMouseDown_ = function(
+  /* eslint-disable no-unused-vars */ e /* eslint-enable no-unused-vars */) {
```

Comment:
If eslint lets you, indent by two more spaces here and everywhere else where you're doing this.

Reply:
Done